listenbrainz/db/user_timeline_event.py

```diff
@@ -79,6 +79,24 @@ def create_user_notification_event(user_id: int, metadata: NotificationMetadata)
         metadata=metadata
     )
 
+def delete_user_recommendation_notification_event(
```

Comment:
This function name seems poorly named, since it can delete more than just a recommendation event.

Reply:
It was supposed to represent the fact that it deletes both recommendation and notification. But I can see how `recommendation_notification` maybe read as one entity. What do you suggest me to name it to? I realised that all methods in `listenbrainz-server/listenbrainz/db/user_timeline_event.py` that deal with only recommendation or notifications, have very generic names (like `create_user_timeline_event`). So should I just name it `delete_user_timeline_event`?